### ISSUE TITLE
PRVHASH 4.3.1

### ISSUE THREAD

**Issue by avaneev** [rurban/smhasher#249]
Hello! I've released prvhash 4.3.1 - it features slightly lower short-message timings in `prvhash64_64m` function, and a higher large-block hashing performance in `prvhash64s`. Functions' outputs did not change.

https://github.com/avaneev/prvhash

**Comment by avaneev** [comment]
Oh well, I do not care about SMHasher presence anymore, there were no further prvhash progress in the recent year, and you may well mark the newer version as not passing tests, I do not know.

**Comment by dumblob** [comment]
> Oh well, I do not care about SMHasher presence anymore

But we do :wink:.

**Comment by rurban** [comment]
Adding latest 4.3.2, looks like nothing much changed.

**Comment by avaneev** [comment]
Yes, the function did not change, and this version only features micro-optimizations. May not be even noticeable in hash-map uses. prvhash64s should be a tiny bit faster.
